docs/api/tensorflow_usage.rst

```diff
@@ -14,19 +14,32 @@ All configuration options are specified using environment variables as shown in
 Optimizing CPU Performance
 --------------------------
 
+.. note::
+    It is advisable to configure these options only if you are an advanced tensorflow user and understand
+    how different operations are implemented under the hood.
+
+These options are useful if you have implemented a custom component as part of your pipeline. Based on the tensorflow operations
```

Comment:
let's rephrase this to clarify that you should know the workings of an implementation well to know whether these configurations make sense - whether that's their own custom implementation or whether they've figured out our implementation